### PR TITLE
Pass --noupdatelatest flag to ci build jobs

### PR DESCRIPTION
**Pull request by jkaniuk**
Needed for presubmit build jobs.
latest*.txt files should be updated only in postsubmits

/cc @mm4tt @krzyzacy @fejta

### CODE REVIEW

File: scenarios/kubernetes_build.py [hunk 1]
```diff
@@ -159,6 +161,8 @@ def main(args):
         '--extra-publish-file', help='Additional version file uploads to')
     PARSER.add_argument(
         '--allow-dup', action='store_true', help='Allow overwriting if the build exists on gcs')
+    PARSER.add_argument(
+        '--noupdatelatest', action='store_true', help='Do not update the latest file')
```
Comment (fejta) [inline scenarios/kubernetes_build.py]: Can we avoid the double negative here? `--skip-update` seems like a better name for this flag.

Comment (mm4tt) [reply scenarios/kubernetes_build.py]: +1

Comment (jkaniuk) [reply scenarios/kubernetes_build.py]: If you insist, I will change it, however I do not see much point in changing the name of the flag that is just passed to underlying tool

Comment (fejta) [reply scenarios/kubernetes_build.py]: The tool should be fixed too (which is outside the scope of this PR)

**Pull request merged**